Web Development/README.md

```diff
@@ -348,6 +348,11 @@ to everyone - everywhere!</td>
         <td>Hitesh Choudhari is a YouTube channel that offers various courses on programming languages. It is one of the best channel to learn web development at free of cost.
         </td>
       </tr>
+      <tr>
+        <td><a href="">https://www.youtube.com/watch?v=bMknfKXIFA8</a>FreeCodeCamp</td>
```

Comment:
Hello @RitaDee It looks good but the problem is that freeCodeCamp youtube channel exist already, change it to something else and then I will merged once youre done.

```suggestion
        <td><a href="">https://www.youtube.com/watch?v=bMknfKXIFA8</a>FreeCodeCamp</td>
```